tools/report-converter/docs/integration-of-kernel-analyzers.md

```diff
@@ -0,0 +1,31 @@
+`report-converter` tools are used to analyze the kernel. Some of the supported parsers are as follows:
+
+## Table of Contents
+
+* [Coccinelle](#coccinelle)
+
+## [Coccinelle](https://github.com/coccinelle/coccinelle)
+[Coccinelle](https://github.com/coccinelle/coccinelle) allows programmers to easily 
+write some complex style-preserving source-to-source transformations on C source code, 
+like for instance to perform some refactorings.
+
+The recommended way of running Coccinelle is to redirect the output to a file and
+give this file to the report converter tool.
+
+The following example shows you how to run Coccinelle and store the results
+found by Coccinelle to the CodeChecker database.
+```sh
+# Change Directory to your project
+cd path/to/your/project
```

Comment:
Well, that is not true. It must be the path to the linux kernel repository other 'make coccicheck' will not exist.